### PR TITLE
Feature/selecting

### PR DESCRIPTION
**Pull request by pattaravit**

### CODE REVIEW

File: app/src/main/java/com/burhanrashid52/photoeditor/PhotoEditor.kt [hunk 1]
```diff
@@ -249,22 +251,20 @@ class PhotoEditor private constructor(builder: Builder) : BrushViewChangeListene
...
-                                if (viewType == ViewType.DRAWING) {
-                                    R.drawable.rounded_border_shape
-                                } else {
-                                    R.drawable.rounded_border_tv
-                                }
-                            }
-                    )
-                    photoEditorCloseButton.takeShowGone { !isBackgroundVisible }
+                    if (photoEditorFrameBorder.tag == true) {
+                        unselectedView()
```
Comment (ademgunay) [inline app/src/main/java/com/burhanrashid52/photoeditor/PhotoEditor.kt]: small type `unselectView()`

File: app/src/main/java/com/burhanrashid52/photoeditor/PhotoEditor.kt [hunk 2]
```diff
@@ -45,6 +46,7 @@ class PhotoEditor private constructor(builder: Builder) : BrushViewChangeListene
     private val imageView: ImageView?
     private val deleteView: View?
     private var currentDrawingView: DrawingView? = null
+    private var currentSelectingView: View? = null
```
Comment (ademgunay) [inline app/src/main/java/com/burhanrashid52/photoeditor/PhotoEditor.kt]: typo `currentSelectedView`

File: app/src/main/java/com/burhanrashid52/photoeditor/PhotoEditor.kt [hunk 3]
```diff
@@ -276,15 +276,40 @@ class PhotoEditor private constructor(builder: Builder) : BrushViewChangeListene
...
+
+    fun unselectedView() {
+        this.currentSelectingView?.apply {
+            photoEditorFrameBorder.tag = false
+            photoEditorFrameBorder.setBackgroundResource(0)
+        }
+        this.currentSelectingView = null
+    }
+
+    fun deleteSelectingView() {
```
Comment (ademgunay) [inline app/src/main/java/com/burhanrashid52/photoeditor/PhotoEditor.kt]: `deleteSelectedView()`

File: app/src/main/java/com/burhanrashid52/photoeditor/PhotoEditor.kt (hunk 2 above)
Comment (pattaravit) [reply app/src/main/java/com/burhanrashid52/photoeditor/PhotoEditor.kt]: ok thx

File: app/src/main/java/com/burhanrashid52/photoeditor/PhotoEditor.kt (hunk 1 above)
Comment (pattaravit) [reply app/src/main/java/com/burhanrashid52/photoeditor/PhotoEditor.kt]: ok

**Pull request merged**